server/service/src/sync/test/test_data/master_list.rs

```diff
@@ -43,6 +46,43 @@ const MASTER_LIST_2: (&'static str, &'static str) = (
 }"#,
 );
 
+const MASTER_LIST_3: (&'static str, &'static str) = (
+    "program_test",
+    r#"{
+    "ID": "program_test",
```

Comment:
```suggestion
    "ID": "program_test_id",
```

Differentiates from description...

Reply:
Above comment and this comment will cause tests to fail. The id is program.id + name_tag.id... So if I change to the above and use the 'description' then change the id in the mock data to what you have suggested above, they will mismatch.